### PR TITLE
(IAC-1449) - Removal of Inappropriate Terminology

### PR DESCRIPTION
**Pull request by david22swan**
Removal of any offensive or inappropriate terminology from the code base.

### CODE REVIEW

File: src/internal/functions/Get-ReadmeContent.ps1 [hunk 1]
```diff
@@ -42,7 +42,7 @@ function Get-ReadmeContent {
   Begin {
     $BuilderModuleGalleryUri      = 'https://www.powershellgallery.com/packages/puppet.dsc'
     $BuilderModuleRepository      = 'https://github.com/puppetlabs/Puppet.Dsc'
-    $BaseProviderSource           = 'https://github.com/puppetlabs/ruby-pwsh/blob/master/lib/puppet/provider/dsc_base_provider.rb'
+    $BaseProviderSource           = 'https://github.com/puppetlabs/ruby-pwsh/blob/main/lib/puppet/provider/dsc_base_provider.rb'
```
Comment (pmcmaw) [inline src/internal/functions/Get-ReadmeContent.ps1:45]: This url doesn't exist but this one does: https://github.com/puppetlabs/ruby-pwsh/blob/main/lib/puppet/provider/dsc_base_provider/dsc_base_provider.rb

Maybe get input from @michaeltlombardi

**Review by pmcmaw — CHANGES_REQUESTED** [review]
1 minor comment.

File: src/internal/functions/Get-ReadmeContent.ps1 (hunk 1 above)
Comment (david22swan) [reply src/internal/functions/Get-ReadmeContent.ps1]: Have updated, Will check with @michaeltlombardi when he gets online

**Review by michaeltlombardi — APPROVED** [review]

File: src/internal/functions/Get-ReadmeContent.ps1 (hunk 1 above)
Comment (michaeltlombardi) [reply src/internal/functions/Get-ReadmeContent.ps1]: Awesome catch, y'all!

**Pull request merged**